docs/kb/use-functions.md

```diff
@@ -134,19 +134,18 @@ To verify the function deployment was successful, click on the `Functions` tab a
     "VC_PASSWORD" : "FillMeIn"
 }
 ```
-Then create the secret in OpenFaaS with this command:
+Then create the Kubernetes secret with this command:
```

Comment:
IMHO the whole section on `secrets` need to be moved. Currently its listed under the verification section.

Suggestion: create a separate section "Knative Function Deployment with Secrets" and describe how to create a secret **before** deploying a function via `kubectl` or `UI`. Also advise on the change in `function.yaml` respectively steps in the UI to attach a secret.

Reply:
Updated